### PR TITLE
DM-3582: corrected and improved the measurement request operation docs

### PR DESCRIPTION
**Pull request by l2p-**
I have updated and corrected the recently added device integration docs about the measurement request operation.

I hope the new file will be integrated automatically and the changed link will work like this

### CODE REVIEW

File: content/device-integration/fragment-library-bundle/measurement-request.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
```
[the remark this answers is not in the document]
Comment (l2p-) [reply content/device-integration/fragment-library-bundle/measurement-request.md:1]: the measurements tab and the get measurements request are not related at all

[the remark this answers is not in the document]
Comment (dafkk) [reply content/device-integration/fragment-library-bundle/measurement-request.md:1]: Screenshot 2024-04-24 at 10 36 02

But now you have two measurement tabs in the docs, that is the problem.

[the remark this answers is not in the document]
Comment (l2p-) [reply content/device-integration/fragment-library-bundle/measurement-request.md:1]: fixed

File: content/device-integration/fragment-library-bundle/measurement-request.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+weight: 130
+title: Manual status update
+layout: bundle
+section: 
+  - device_management
+---
+
+**Manual status update**
```
Comment (BeateRixen) [inline content/device-integration/fragment-library-bundle/measurement-request.md:9]: As this is already the title we should remove this repetition here.

Comment (BeateRixen) [inline content/device-integration/fragment-library-bundle/measurement-request.md:9]:
```suggestion
```

**Review by BeateRixen — APPROVED** [review]

**Review by dafkk — APPROVED** [review]

**Pull request merged**